storage/storage.go

```diff
@@ -21,6 +21,9 @@ var (
 	ErrNoObjectFound = fmt.Errorf("no object found")
 )
 
+// FollowSymlinks is used to check the value of --no-follow-symlinks flag globally
+var FollowSymlinks = true
```

Comment:
instead of a global, we can pass `followSymlink` to storage.List method.

Reply:
I think `symbolic links` is not a part of the storage abstraction. I do not prefer adding `followSymlink` parameter to  the interface. I am thinking of a better way than using a global variable.

Reply:
why not? symlink is actually a part of the storage. some storage systems have support, some don't.

An object's type could be a symlink, or a file, or a directory.
List method could have support for following symlinks, or visiting directories.

Reply:
Because I didn't want to add another empty implementation like `storage.MakeBucket`. 
https://github.com/peak/s5cmd/blob/1c8d8274e1f44f9c17312877e3e70060032dfcc3/storage/fs.go#L190

Reply:
I don't follow. We don't need any method or an empty implementation. S3 will ignore the flag, Filesystem storage will pass it to the godirwalk.Walk. Do I miss something?

Reply:
Tried your suggestion, updated the PR.  It is better than having a global variable. thanks!

Reply:
🚀